packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.types.ts

```diff
@@ -239,6 +239,11 @@ export interface IGroupRenderProps {
    * @defaultvalue false
    */
   showEmptyGroups?: boolean;
+
+  /**
+   * String that allows for an aria-label to be passed in
+   */
+  ariaLabelForToggleAllGroupsButton?: string;
```

Comment:
Why do we need this aria label when the toggle all groups button is managed by the `DetailsHeader`? If the intent was to enable passing an `aria-label` prop to the collapse/expand button on each `GroupHeader` then we already have it enabled by the `headerProps` in `IGroupRenderProps` interface through `IGroupHeaderProps`[expandButtonProps].

Reply:
This isn't the group header. This is for the overall list header which controls all the groups.